scripts/gen_eval_files.lua

```diff
@@ -315,6 +327,26 @@ local function render_api_meta(_f, fun, write)
     end
   end
 
+  -- LuaLS doesn't support @note. Render @note items as a markdown list.
+  if fun.notes and #fun.notes > 0 then
+    -- local notes = 'Note:\n'
+    write('--- Note:')
+    for _, note in ipairs(fun.notes) do
+      note = text_utils.md_to_vimdoc(' - ' .. note, 0, 0, 74)
+      for _, l in ipairs(split(vim.trim(norm_text(note)))) do
+        write('--- ' .. l:gsub('\n*$', ''))
```

Comment:
Currently abusing `md_to_vimdoc` here since we don't have something like `md_to_docstring`, or some other way to format/fit markdown text into the rendered docstring. cc @lewis6991 for sanity check

Reply:
What format is docstring? I think using md in doc comments is an acceptable format.

Reply:
by "docstring" i mean md format for the `_meta/*.lua` files. I didn't find a function that lets me format for that in markdown format instead of "vimdoc" format (`md_to_vimdoc`).